test/modules/slackIntegration.spec.js

```diff
@@ -91,6 +90,7 @@ describe('Slack Integration Test Suite', async () => {
     );
     expect(validChannel).to.equal(exists);
   });
+  
```

Comment:
There is really no change in this file except the addition of white spaces. Please remove the white spaces added because it's making your pull request unnecessarily large.

Reply:
I have removed the whitespace